isPlandromRecursive.py

```diff
@@ -0,0 +1,24 @@
+# Define a recursive procedure is_palindrome, that takes as input a string, and returns a
+# Boolean indicating if the input string is a palindrome.
+
+# Base Case: '' => True
+# Recursive Case: if first and last characters don't match => False
+# if they do match, is the middle a palindrome?
+
+def is_palindrome(s):
+    if s == '':
+        return  True
+    if s[0] != s[len(s)-1]:
```

Comment:
In python; we can also use negative index.
-1 > Last element
-2 >2nd last element
and so on
Hence, you can check `if s[0] != s[-1]` 
This [link to Python Syntax](https://learnxinyminutes.com/docs/python/) can be helpful for you.